core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -292,12 +292,8 @@ object ConsumerGroupCommand extends Logging {
       }
 
       getLogEndOffsets(topicPartitions).map {
-        logEndOffsetResult =>
-          logEndOffsetResult._2 match {
-            case LogOffsetResult.LogOffset(logEndOffset) => getDescribePartitionResult(logEndOffsetResult._1, Some(logEndOffset))
-            case LogOffsetResult.Unknown => getDescribePartitionResult(logEndOffsetResult._1, None)
-            case LogOffsetResult.Ignore => null
```

Comment:
As far as I can tell, this was the only use of `LogOffsetResult.Ignore`. Maybe we can get rid of it? 

If we did that, then we could probably also get rid of `LogOffsetResult` and replace it with a simple `Option[Long]`, but we can leave that for a separate PR.